id-repository-default.properties

```diff
@@ -427,4 +427,13 @@ mosip.role.idrepo.vid.postvidreactivate=RESIDENT,ID_REPOSITORY
 mosip.role.idrepo.vid.postdraftvid=REGISTRATION_PROCESSOR,ID_REPOSITORY
 mosip.mask.function.identityAttributes=convertToMaskDataFormat
 
-mosip.role.idrepo.credentialrequest.postv2requestgeneratorrid=CREDENTIAL_REQUEST,ID_REPOSITORY
\ No newline at end of file
+mosip.role.idrepo.credentialrequest.postv2requestgeneratorrid=CREDENTIAL_REQUEST,ID_REPOSITORY
+
+mosip.idrepo.credential.request.enable-convention-based-id=true
+mosip.idrepo.credential-request-v2.rest.uri=${mosip.idrepo.credrequest.generator.url}/v1/credentialrequest/v2/requestgenerator/{rid}
+mosip.idrepo.credential-request-v2.rest.httpMethod=POST
+mosip.idrepo.credential-request-v2.rest.headers.mediaType=application/json
+mosip.idrepo.credential-request-v2.rest.timeout=100
+
+mosip.idrepo.identity.disable-uin-based-credential-request=false
```

Comment:
let us add a comment on this property as below
"It is recommended to not support UIN based authentication (both external & internal), To stop issuing UIN+credential to IDA set flag to true otherwise set the flag to false to issue UIN+credential to IDA."